stateful-datatable-product/README.md

```diff
@@ -1,21 +1,23 @@
 # Stateful Datatable
 
-An utility Dialogue to implement a stateful-datatable with Ivy and Primefaces.
+A utility Dialog to implement a stateful-datatable with Ivy and Primefaces.
 
 ## Demo
 
 In this demo, a lazy datatable is listing all the business objects available in the data base or in the Ivy business repo.
 
+![List of Products](DemoCapture.PNG "List of Products")
+
 ## Setup
 
  - Before starting, a data base needs to be setup and then generate the schema from the persistence unit.
- - Next, start the test process: "Create Test Data Database" to generate the test data for the demo.
+ - Next, start the test process: ``Create Test Data Database`` to generate the test data for the demo.
 
- - If you need to start the stateful-datatable with Ivy business repo, no need to setup the data base but you only need to start the test process: "Create Test Data Repo".
+ - If you need to start the stateful-datatable with Ivy business repo, no need to setup the data base but you only need to start the test process: ``Create Test Data Repo``.
 
 ## How to add a colum:
 
-The place each step takes place has a comment like "Adding columns STEP 1", "Adding columns STEP 2" and so on.
+The place each step takes place has a comment like ``Adding columns STEP 1``, ``Adding columns STEP 2`` and so on.
```

Comment:
just one of these quotes at the start and one at the end ... I don't think that proper markdown right now.
If you are struggling with markdown: here's a pretty effective markdown editor I like to use http://pad.haroopress.com/user.html